src/testUtils/mocks/entities.ts

```diff
@@ -810,6 +811,11 @@ const defaultDividendDistributionOptions: DividendDistributionOptions = {
     remainingFunds: new BigNumber(100),
     fundsReclaimed: false,
   },
+  getParticipant: {
+    identity: ('someIdentity' as unknown) as Identity,
```

Comment:
You can remove this line, and in `configureDividendDistribution` you use `mockInstanceContainer.identity` when merging this with the passed options (see how it's done in `configureTickerReservation` for an example)